PegSolitaire.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <iostream>
+#include <chrono>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <array>
+#include <unordered_set>
+#include <tuple>
+#include <regex>
+
+
+enum move {
+	up,
+	right,
+	down,
+	left
+};
+static class ps {

```

Comment:
Does not compile under gcc:
static is not permitted under declaration of a type
```suggestion
class ps {
```